services/release/release.go

```diff
@@ -297,10 +297,11 @@ func DeleteReleaseByID(ctx context.Context, id int64, doer *user_model.User, del
 	}
 
 	if delTag {
-		if stdout, err := git.NewCommand(ctx, "tag", "-d", rel.TagName).
+		stdout := new(strings.Builder)
+		if err := git.NewCommand(ctx, "tag", "-d", rel.TagName).
 			SetDescription(fmt.Sprintf("DeleteReleaseByID (git tag -d): %d", rel.ID)).
-			RunInDir(repo.RepoPath()); err != nil && !strings.Contains(err.Error(), "not found") {
-			log.Error("DeleteReleaseByID (git tag -d): %d in %v Failed:\nStdout: %s\nError: %v", rel.ID, repo, stdout, err)
+			RunWithContext(&git.RunContext{Dir: repo.RepoPath(), Timeout: -1, Stdout: stdout}); err != nil && !strings.Contains(err.Error(), "not found") {
```

Comment:
Her i got
```
Stdout:
Error: exit status 1
```
Is this if statement is valid `err != nil && !strings.Contains(err.Error(), "not found")`

Reply:
@martinscholz83 take https://github.com/go-gitea/gitea/pull/18978/commits/3305f5a6f52fc29ab58216b121dd621ab5e29604 as example

Reply:
Got it. thx!